vignettes/benchmarking.Rmd

```diff
@@ -0,0 +1,346 @@
+---
+title: "D. Benchmarking with ProteinGym"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{D. Benchmarking with ProteinGym}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+    collapse = TRUE,
+    comment = "#>"
+)
+```
+
+Original version: 2 October, 2024
+
+```{r setup, message = FALSE}
+library(AlphaMissenseR)
+library(dplyr)
+library(purrr)
+library(tidyr)
+library(forcats)
+library(ggdist)
+library(gghalves)
```

Comment:
I think that purrr, forcats, ggidst, and gghalves are used kind of sparingly in the vignette, and it might help to signal these with, e.g., `purrr::pluck()`. But actually with respect to purrr and forcats, I wonder if it is more straight-forward to use base R functionality and skip the extra dependencies? I for one am not familiar enough with these packages to know what exactly the code is doing, so it is an impediment to my understanding...